docs/developer-guide/ksqldb-reference/scalar-functions.md

```diff
@@ -979,19 +979,19 @@ Given any ksqlDB type, returns the equivalent JSON string.
 ```sql title="Primitives types"
 TO_JSON_STRING(1) => "1"
 TO_JSON_STRING(15.3) => "15.3"
-TO_JSON_STRING("abc") => "\"abc\""
-TO_JSON_STRING(true) => "true"
-TO_JSON_STRING(2021-10-11) => DATE type, "\"2021-10-11\""
-TO_JSON_STRING(13:25) => TIME type, "\"13:25:10\""
-TO_JSON_STRING(2021-06-31T12:18:39.446) => TIMESTAMP type, "\"2021-06-31T12:18:39.446\""
-TO_JSON_STRING(NULL) => "null"
+TO_JSON_STRING('abc') => '\"abc\"'
+TO_JSON_STRING(true) => 'true'
+TO_JSON_STRING(2021-10-11) => DATE type, '\"2021-10-11\"'
```

Comment:
Output in this case shows up as:
```
2000
```
I assume this is a numeric representation of the date 2021-10-11